Search our custom $(LIB_DIRS) path before the stuff 
gtk-config injects in $(LIBS).

http://bugs.gentoo.org/94867

--- jscalibrator/Makefile
+++ jscalibrator/Makefile
@@ -132,5 +132,5 @@
 modules: $(OBJ_C) $(OBJ_CPP)
 	@echo  -n "Linking modules..."
-	@$(CPP) $(OBJ_C) $(OBJ_CPP) -o $(BIN) $(LIBS) $(LIB_DIRS)
+	@$(CPP) $(OBJ_C) $(OBJ_CPP) -o $(BIN) $(LIB_DIRS) $(LIBS)
 	@echo -n "   "
 	@-$(LS) $(LSFLAGS) $(BIN)
